src/Zicht/Bundle/SolrBundle/Command/ReindexCommand.php

```diff
@@ -22,6 +22,12 @@
 
 /**
  * Reindex a specified repository or entity in SOLR
+ *
+ * Calls an Update Query for Entities by default.
+ * If a Entity is of the type Extractable the indexer tries first to call the Extract Query.
+ * The Extractable interface has an optional resource that's why we fall back to Update in case of missing
+ * a resource. The Update Query is much cheaper in that context and can be batched. The Extract is not batchable
+ * but for convenience it is enabled in this command.
```

Comment:
Dus het kan niet in een batch plaatsvinden, maar het wordt hier wel geprobeerd?  Dan zou ik eerder een foutmelding oid verwachten.  Of nog veel liever, dat de code alle updates in een batch doet, maar alle extracts 1 voor 1.